reader-osm/src/test/java/com/graphhopper/routing/RoutingAlgorithmWithOSMTest.java

```diff
@@ -664,8 +668,8 @@ Graph runAlgo(TestAlgoCollector testCollector, String osmFile,
             hopper.importOrLoad();
 
             Collection<AlgoHelperEntry> prepares = createAlgos(hopper, queryProfile);
-            FlagEncoder encoder = hopper.getEncodingManager().getEncoder(queryProfile.getName());
-            EdgeFilter edgeFilter = DefaultEdgeFilter.allEdges(encoder.getAccessEnc());
+            BooleanEncodedValue inSubnetworkEnc = hopper.getEncodingManager().getBooleanEncodedValue(InSubnetwork.key(queryProfile.getName()));
+            EdgeFilter edgeFilter = new FiniteWeightFilter(hopper.createWeighting(queryProfile, new PMap()), inSubnetworkEnc);
```

Comment:
This test is kind of a pain, because it partly replicates the real routing code in GraphHopper... Do we need inSubnetwork here? Probably yes, because we are running on real OSM data and not on simple sample graphs?

Reply:
-> fixed in #2293